pkg/controller/controller.go

```diff
@@ -752,6 +752,10 @@ func (c *Controller) Run(ctx context.Context) {
 		util.LogFatalAndExit(err, "failed to set NB_Global option ls_ct_skip_dst_lport_ips")
 	}
 
+	if err := c.OVNNbClient.SetNodeLocalDNSIP(c.config.NodeLocalDNSIP); err != nil {
+		util.LogFatalAndExit(err, "failed to set NB_Global option node_local_dns_ip")
+	}
```

Comment:
Validate `c.config.NodeLocalDNSIP` before setting the ovn nb option .

Reply:
1